### DISCUSSION TITLE
Hide from Mac top menu bar

### DISCUSSION THREAD

**Discussion by Jashun44** [stashapp/stash#4724] [Q&A] [answered]
Sorry if this is a dumb question... Is there any way to hide the app from the top menu bar on a Mac? I'd rather it not be noticed up there, showing its existence, Lol. I couldn't find an option in the settings.  Thanks!

**Comment by DogmaDragon** [comment, marked as answer]
I'm not familiar with macOS, but you can run command-line app which doesn't support menu bar. https://docs.stashapp.cc/installation/macos/#command-line-app

Maybe this can help https://www.makeuseof.com/how-to-remove-menu-bar-items-mac/
